packages/model-viewer/scripts/ci-run-tests.sh

```diff
@@ -20,11 +20,8 @@ set -x
 
 if [ "${TEST_TYPE}" = "unit" ]; then
 
-  npm run test
+  npm run wct-local-chrome
 
-  if  [[ "${TRAVIS_PULL_REQUEST}" = "false" && "${TRAVIS_BRANCH}" != "master" ]]; then
```

Comment:
Does this mean third parties start browserStack tests, or did this logic end up elsewhere?

Reply:
This got moved elsewhere